stdpopsim/catalog/SusScr/species.py

```diff
@@ -0,0 +1,150 @@
+import stdpopsim
+
+from . import genome_data
+
+_ZhangEtAl = stdpopsim.Citation(
+    doi="https://doi.org/10.1016/j.gpb.2022.02.001",
+    year=2022,
+    author="Mingpeng Zhang et al.",
+    reasons={
+        stdpopsim.CiteReason.MUT_RATE,
+        stdpopsim.CiteReason.GEN_TIME,
+        stdpopsim.CiteReason.POP_SIZE,
+    },
+)
+_JohnssonEtAl = stdpopsim.Citation(
+    doi="https://doi.org/10.1186/s12711-021-00643-0",
+    year=2021,
+    author="Martin Johnsson et al.",
+    reasons={stdpopsim.CiteReason.REC_RATE},
+)
+# [The following are notes for implementers and should be deleted
+#  once the recombination rates have been inserted]
+# This is the per-chromosome recombination rate, typically the mean
+# rate along the chromosome.
+# Values in this dictionary are set to -1 by default, so you have
+# to update each one. These should be derived from the most reliable
+# data and how they were obtained should be documented here.
+# The appropriate citation must be added to the list of
+# recombination_rate_citations in the Genome object.
+
+_recombination_rate = {
+    "1": 5.335247608983185e-09,
+    "2": 8.6057260999254e-09,
+    "3": 9.751141303348547e-09,
+    "4": 9.79460049114614e-09,
+    "5": 1.1992632296111e-08,
+    "6": 8.786173521918499e-09,
+    "7": 1.0887772285689678e-08,
+    "8": 8.923031414594611e-09,
+    "9": 9.337506174170301e-09,
+    "10": 1.654233205589542e-08,
+    "11": 1.2145120574910172e-08,
+    "12": 1.6905062453467094e-08,
+    "13": 6.162796630328349e-09,
+    "14": 8.721569375548237e-09,
+    "15": 8.114468762954893e-09,
+    "16": 1.1407464686635653e-08,
+    "17": 1.3779949516098884e-08,
+    "18": 1.3679923888648167e-08,
+    "X": -1,
```

Comment:
We should provide a value for X chromosome too. I appreciate that Johnsson et al. (2021) does not list X-chromosome recombination rate. Hence, I suggest that you calculate average recombination rate across other chromosomes (exclude Y and MT) and replace this for the X-chromosome rec rate value. Make this calculation clear in the code here.